### PR TITLE
done the calculate working days.

### PR DESCRIPTION
**Pull request by AnastasiaMamontova**
use the method addEventListener instead of oninput

### CODE REVIEW

File: Tasks/4 - Input Field/ScriptWorkdays.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function calculateWorkingDays() {
```
Comment (gologames) [inline Tasks/4 - Input Field/ScriptWorkdays.js]: The filename `ScriptWorkdays.js` is written in what is called **PascalCase** style. PascalCase is commonly used for React applications, but it is not typical for vanilla JavaScript code. For this current situation, it is better to use **kebab-case**, e.g., `script-workdays.js`

But it is better to remove the `Script` prefix and just name the file `workdays.js`

File: Tasks/4 - Input Field/ScriptWorkdays.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  var daysInYearInput = document.getElementById("daysInYear");
```
Comment (gologames) [inline Tasks/4 - Input Field/ScriptWorkdays.js]: Please, use `const` instead of `var`

File: Tasks/4 - Input Field/ScriptWorkdays.js (hunk 1 above)
Comment (AnastasiaMamontova) [reply Tasks/4 - Input Field/ScriptWorkdays.js]: done

File: Tasks/4 - Input Field/ScriptWorkdays.js (hunk 2 above)
Comment (AnastasiaMamontova) [reply Tasks/4 - Input Field/ScriptWorkdays.js]: done

File: Tasks/4 - Input Field/workdays.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Working Day Calculator</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="ScriptWorkdays.js"></script>
```
Comment (gologames) [inline Tasks/4 - Input Field/workdays.html]: But link to script still has old name

Comment (AnastasiaMamontova) [reply Tasks/4 - Input Field/workdays.html]: done

**Review by gologames — APPROVED** [review]

**Pull request merged**